drivers/rtc/rtc-pcf2127.c

```diff
@@ -569,6 +569,7 @@ static int pcf2127_probe(struct device *dev, struct regmap *regmap,
 {
 	struct pcf2127 *pcf2127;
 	int ret = 0;
+	bool ts_off = true;
```

Comment:
Nit: Reverse christmas tree.

Reply:
So how is ts_off ever set to false? Shouldn't this be set through a DT property or something?

Reply:
For now I disable it without any option to override. As I don't know how this will be done in mainline later. Thus I don't need to change anything besides the driver for now.